gammapy/utils/nddata.py

```diff
@@ -0,0 +1,441 @@
+"""Scratch for NDDataArray class"""
```

Comment:
* Missing license and future import line. Copy them from any other file in Gammapy.
* Missing `__all__` and you should add an `automodapi` directive for this file below the one [here](https://github.com/gammapy/gammapy/blame/master/docs/utils/index.rst#L118) so that these classes show up in the API docs.
* Can you put a better module-level docstring?